go/ct/common/immutable_hash_array.go

```diff
@@ -64,12 +64,12 @@ func (h *ImmutableHashArray) UnmarshalJSON(data []byte) error {
 	// Unmarshal the JSON array into a slice of tosca.Hash
 	var hashes [256]tosca.Hash
 	err := json.Unmarshal(data, &hashes)
-	if err != nil {
+	if err != nil { // coverage-ignore
```

Comment:
We should use `// coverage-ignore` very sparsely, and if we do, with a proper explanation.